feature/experimental/isis/ate_tests/lsp_updates_test/lsp_updates_test.go

```diff
@@ -62,7 +73,7 @@ func TestOverloadBit(t *testing.T) {
 		GetOrCreateOverloadBit().SetBit = ygot.Bool(true)
 	ts.PushDUT(context.Background())
 	// TODO: Verify the link state database once device support is added.
-	if err := check.Equal[uint32](overloads.State(), 1).AwaitFor(time.Second*10, ts.DUTClient); err != nil {
+	if err := check.Equal(overloads.State(), uint32(olVal+1)).AwaitFor(time.Second*10, ts.DUTClient); err != nil {
```

Comment:
nit: the overload bit can either be 1 or 0, right? Not sure why we want `+1` here (instead of just saying 1)?

Reply:
This leaf /network-instances/network-instance/protocols/protocol/isis/levels/level/system-level-counters/state/database-overloads of type uint32 returns number of times is-is database is in overload state.
Here we are checking that after we set the overload bit the counter value should be initial value +1.